src/ci/github-actions/jobs.yml

```diff
@@ -220,6 +220,12 @@ auto:
   - name: dist-x86_64-netbsd
     <<: *job-linux-4c
 
+  - name: dist-x86_64-solaris
+    <<: *job-linux-4c
+
+  - name: dist-sparcv9-solaris
+    <<: *job-linux-4c
```

Comment:
can you run `[at]bors try`?
Otherwise I can do it to test this 👍

Reply:
@bors try